mobile/lib/screens/dashboard/dashboard_view.dart

```diff
@@ -134,11 +105,12 @@ class _DashboardViewState extends State<DashboardView>
                       );
 
                       return Expanded(
-                        child: Showcase(
-                          key: _favoritesShowcaseKey,
+                        child: CustomShowcaseWidget(
+                          showcaseKey: _favoritesShowcaseKey,
+                          descriptionHeight: 120,
                           description:
-                              'Find the latest air quality from your favorite locations',
-                          child: DashboardTopCard(
+                              "Find the latest air quality from your favorite locations",
+                          childWidget: DashboardTopCard(
```

Comment:
for the sake of naming convention, use `child` instead of `childWidget`

Reply:
Done